src/components/transactions/TxInfo/index.tsx

```diff
@@ -29,11 +29,13 @@ export const TransferTx = ({
   omitSign = false,
   withLogo = true,
   preciseAmount = false,
+  truncateText = false,
```

Comment:
<TokenAmount/> component is being used in various pages

truncate text is applied specifically to <TokenAmount/> in transaction info component , to avoid unindented UI changes in other pages

Reply:
Didn't we decide to truncate text everywhere in https://github.com/safe-global/safe-wallet-web/pull/4034#discussion_r1721438178?